templates/admin/style.css.ts

```diff
@@ -3,7 +3,9 @@ import { readFile, TemplateFunction } from "../../src/lib/createAdapter";
 export = (answers => {
 
 	const isAdapter = answers.features.indexOf("adapter") > -1;
-	if (!isAdapter) return;
+	const useJsonConfig = answers.adminUi === 'json';
+	const noConfig = answers.adminUi === 'none';
+	if (!isAdapter || useJsonConfig || noConfig) return;
```

Comment:
I think we can simplify this to html only, right?

Reply:
HTML and React